tests/appsec/iast/source/test_iast_request_body.py

```diff
@@ -0,0 +1,27 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+import pytest
+import re
+from utils import weblog, interfaces, context, coverage, released, missing_feature
+
+
+if context.library == "cpp":
+    pytestmark = pytest.mark.skip("not relevant")
+
+
+@coverage.basic
+@released(dotnet="?", golang="?", php_appsec="?", python="?", ruby="?")
+@released(java={"spring-boot": "1.7.0", "*": "?"})
```

Comment:
Can we enable it for some other variants?

Reply:
I will create another PR fixing the Xpassed test for different springboot scenarios